src/sargon.udl

```diff
@@ -10,4 +10,7 @@ typedef string Timestamp;
 [Custom]
 typedef sequence<i8> BagOfBytes;
 
+[Custom]
+typedef string InnerDecimal;
```

Comment:
Here is the trick, we vendor `Decimal192` as a struct with one single stored property (field) `__inner: InnerDecimal` which in the land of Swift and Kotlin will be `__inner: String`. It is given that name to hide it as much as possible. It is, in fact, NOT a String but the rust type `InnerDecimal` which in Rust land holds a ScryptoDecimal192. 

This is the best compromise, it allwos for usage of a real decimal in Rust land - the ScryptoDecimal192 - and it allows for its OWN type in Swift/Kotlin, and NOT a silly error prone `typealias Decimal192 = String`.